src/mainboard/Makefile.common

```diff
@@ -0,0 +1,67 @@
+# This Makefile is included from src/mainboards/VENDOR/BOARD/Makefile
```

Comment:
This Makefile is included from src/mainboards/VENDOR/BOARD/Makefile
                                             in?
                                             by?

Can you set OREBOOT in here too? 
OREBOOT=$(magic)

Reply:
I don't understand. For example "src/mainboard/Makefile.common is included from src/mainboards/sifive/hifive/Makefile".

What is the purpose of the OREBOOT magic?